addons/ondevice-knobs/src/types/Number.js

```diff
@@ -2,15 +2,37 @@ import PropTypes from 'prop-types';
 import React from 'react';
 import { TextInput, View, Slider } from 'react-native';
 
+const replaceComma = x => x.replace(',', '.');
```

Comment:
What's this for?

Reply:
some keyboards have , instead of . in number keyboard.

Reply:
I mean some languages.

Reply:
Is there some kind of i18n function for that?

Reply:
don't really know. Don't see a big need for it when it is just a simple replace :)

Reply:
Isn't that the point -- use a function by somebody who cares about the minutiae of i18n so you don't have to? Number  pads also have *'s and #'s, why not replace those too?

Reply:
@shilman, Actually I do replace them, since I check for format xxxxx.yyyyyy, if any other  character is entered we ignore it :P Thats why in this case I think replace comma is sufficient :)

Reply:
Can you restructure the code at all so it's more obvious what you're doing? Like a function `parseNumber` and add a few unit tests?

```
expect(parseNumber('1,000,000.24')).toEqual(1000000.24)
expect(parseNumber('1000000,24')).toEqual(1000000.24)
```

Reply:
Now when you wrote an example I see what you mean. I will simply ignore , completely, and we will not allow to enter it.